tests/conftest.py

```diff
@@ -301,22 +306,41 @@ def invoke_command(
     test_context: Optional[Context] = None,
 ) -> Result:
     client = _create_client()
-    out_file_name = "out_file_.json"
-    context = _create_test_context() if test_context is None else test_context
+    out_file_name = "out_file.json"
+
+    if cmd.name == import_artifacts.name:
+        out_args = []
+    elif cmd.name == update.name:
+        out_args = ["-s", out_file_name]
+    else:
+        out_args = ["--out", out_file_name]
+
+    api_url = None
+    if "--api-server" in args:
+        api_server_flag_index = args.index("--api-server")
+        api_url = args.pop(api_server_flag_index + 1)
+        args.remove("--api-server")
+
+    context = test_context
+    if test_context is None:
+        context = _create_test_context(api_url)
```

Comment:
Programming tip: I'd do all of above outside of the helper, and pass only the args and context you want here.

Reply:
All of the above is done inside `invoke_command` as even if you don't test `--out/--save` we want to use it as we save the result of the command in `out_file.json` inside the isolated filesystem and read the result of the command from this file.. 
I want the tests to provide only args that are connected to what they are testing.

Reply:
Something I am investigating now is how to pass `--api-server` as it's an admin option.
Meaning it's easy for me to pass options related to the command I am currently testing for example `--in` and `--out` for `sign` command, but I need to pass `--api-server` to admin - `rstuf admin --api-server <URL> send ...` which I have not found yet how to do and that's why I am settings `settings.SERVER` as a solution for now.

Reply:
>  I am currently testing for example --in and --out for sign command,

How can you test passing `--out`, if your helper always adds it?

Reply:
By the way, changing the context of the command seems to be the way to go as mentioned by a `click` core-developer in two issues: https://github.com/pallets/click/issues/831#issuecomment-671048863, https://github.com/pallets/click/issues/832#issuecomment-671052776.

Reply:
> > I am currently testing for example --in and --out for sign command,
> 
> How can you test passing `--out`, if your helper always adds it?

By not calling my helper. See: https://github.com/repository-service-tuf/repository-service-tuf-cli/blob/0d73c3d7e8059bdbcb8b1722ab871c41c0c85e06/tests/unit/cli/admin/test_ceremony.py#L10

Reply:
Sure, but we were discussing the helper.

Anyway, I have already approved the PR. So feel free to merge.